commit cbd1708f27ac93306f259bc7da7deefa483e01bc (refs/changes/59/4159/3)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2018-06-08T14:13:02+00:00 (1 year, 4 months ago)
    
    TRITON-462 triton services still on nodejs v4 should use the latest point release
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/Makefile b/Makefile
index f6563d5..ab8624f 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@
 #
 
 #
-# Copyright (c) 2016, Joyent, Inc.
+# Copyright (c) 2018, Joyent, Inc.
 #
 
 #
@@ -42,7 +42,7 @@ JSSTYLE_FLAGS	 = -f tools/jsstyle.conf
 SMF_MANIFESTS_IN = smf/manifests/papi.xml.in
 
 
-NODE_PREBUILT_VERSION=v4.6.1
+NODE_PREBUILT_VERSION=v4.9.0
 ifeq ($(shell uname -s),SunOS)
         NODE_PREBUILT_TAG=zone
         # Allow building on other than sdc-minimal-multiarch-lts@15.4.1
